### PR TITLE
VUSB.unplug: ignore exception when domain does not exist

### PR DESCRIPTION
**Pull request by robhoes**
Signed-off-by: Rob Hoes <rob.hoes@citrix.com>

### CODE REVIEW

**Comment by taoyongd** [comment]
Ok，Thanks @robhoes , ignore exception can solve this I think.

File: lib/xenops_server.ml [hunk 1]
```diff
@@ -1761,7 +1761,7 @@ and perform ?subtask ?result (op: operation) (t: Xenops_task.task_handle) : unit
 					Some Needs_unplug
 				end in
 			let operations_of_request = function
-				| Needs_unplug -> Some (Atomic(VBD_unplug (id, true)))
+				| Needs_unplug -> Some (Atomic(VUSB_unplug (id, true)))
```
Comment (mseri) [inline lib/xenops_server.ml]: I think this has only `id`

**Comment by krizex** [comment]
Similar issue and fix with: https://github.com/xapi-project/xen-api/pull/1175

**Comment by robhoes** [comment]
@krizex Indeed :)

**Pull request merged**